### PR TITLE
Allow writing audit log to syslog and systemdlog

### PR DESCRIPTION
**Pull request by nickvergessen**
In case you think we should backport this, be aware that it might change the logging, as audit currently always logs to file and with this patch it starts falling back to the normal logging which could be syslog and others.
We could also change this, but I think it would make the most sense.

Also on merge and after discussion of above issue the documentation in https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html#admin-audit-log needs a treatment

### CODE REVIEW

File: lib/private/Log/Errorlog.php [hunk 1]
```diff
@@ -29,13 +29,20 @@
 
 class Errorlog implements IWriter {
 
+	/** @var string */
+	protected $tag;
+
+	public function __construct(string $tag = 'owncloud') {
```
Comment (skjnldsv) [inline lib/private/Log/Errorlog.php:35]: owncloud?

**Comment by skjnldsv** [comment]
I think I'm lacking too much knowledge to give a proper review sorry :(

**Review by blizzz — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**